### PR TITLE
SW-1226 Rename "Germination Testing" withdrawal purpose

### PR DESCRIPTION
**Pull request by sgrimm**
Use the term "viability testing" in the list of withdrawal purposes to align with
the app terminology.

### CODE REVIEW

**Comment by sgrimm** [comment]
Current dependencies on/for this PR:
* main
  * **PR #502** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/502" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-server/502?utm_source=stack-comment).

File: src/main/resources/db/migration/R__TypeCodes.sql [hunk 1]
```diff
@@ -154,7 +154,7 @@ VALUES (1, 'Propagation'),
        (4, 'Broadcast'),
        (5, 'Share with Another Site'),
        (6, 'Other'),
-       (7, 'Germination Testing')
+       (7, 'Viability Testing')
```
Comment (karthikbtf) [inline src/main/resources/db/migration/R__TypeCodes.sql:157]: @sgrimm what's the process to update `dump/dump.sql` in the FE to capture this change?

Comment (sgrimm) [reply src/main/resources/db/migration/R__TypeCodes.sql:157]: Ah, good question! We should have a script for that, or at least docs. The general process is to restore the DB using this dump file, start the server to let it run migrations, then run `pg_dump` and overwrite the dump file.

You shouldn't _need_ to update the dump file, though; the value will be updated by the database migrations when the server starts up.

**Comment by karthikbtf** [comment]
The build seems to be failing https://github.com/terraware/terraware-server/actions/runs/2693641947 in `generate-and-deploy`
```
found 0 vulnerabilities
/bin/sh: 1: mjml: not found
```

**Comment by sgrimm** [comment]
Probably something to do with moving the MJML rendering to a separate source file. Luckily, this only affects the schema docs generation, not the server code. I'll take a look.

**Pull request merged**